### PR TITLE
[FEATURE] Rewrite for TYPO3 9.x

### PR DESCRIPTION
**Pull request by bmoex**

### CODE REVIEW

File: Classes/Form/Element/SelectorKoningGeo.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+{
+
+    /**
+     * Handler for single nodes
+     *
+     * @return array As defined in initializeResultArray() of AbstractNode
+     */
+    public function render(): array
+    {
+
```
Comment (jp1987) [inline Classes/Form/Element/SelectorKoningGeo.php]: witregel teveel

File: Classes/Form/Element/SelectorKoningGeo.php [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+use KoninklijkeCollective\KoningGeo\Domain\Model\Location;
+use TYPO3\CMS\Backend\Form\Element\InputTextElement;
+use TYPO3\CMS\Core\Database\ConnectionPool;
+use TYPO3\CMS\Core\Database\Query\QueryBuilder;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Core\Utility\MathUtility;
+
+/**
+ * Override value for SelectorKoningGeo
```
Comment (jp1987) [inline Classes/Form/Element/SelectorKoningGeo.php]: witregel?

**Comment by bmoex** [comment]
This breaks the eID functionality; as the DataMap cannot cast the new 'input' type to the required value.. Stupid -_-

**Comment by bmoex** [comment]
#yolo

**Pull request merged**